app/views/exports/index.html.haml

```diff
@@ -4,19 +4,15 @@
   %thead
     %tr
       %th Date
-      %th Sites
-      %th Status
+      %th Percent Complete
       %th Variables
-      %th Email
       %th Actions
   %tbody
     - @exports.each do |export|
       %tr
         %td= export.created_at
-        %td= export.sites.try(:join, ', ')
-        %td= export.status
+        %td= "#{export.progress / 25}%"
```

Comment:
Nevermind, I see there is a bunch in the export process that is causing it to put extremely large values in there.  I'll have another branch that fixes this, in the end the output for this line should look like `number_to_percentage export.progress, precision: 0`